src/Scaffolding/VS.Web.CG.Mvc/BlazorIdentity/BlazorIdentityHelper.cs

```diff
@@ -101,5 +101,23 @@ internal static CodeSnippet[] ApplyIdentityChanges(CodeSnippet[] filteredChanges
 
             return filteredChanges;
         }
+
+        internal static string BlazorIdentityReadmeFileName = "Scaffolding-README.md";
+        internal static string BlazorIdentityReadmeString =
+@"Support for Blazor Identity was added to your project.
+
+For setup and configuration information, see https://learn.microsoft.com/en-us/aspnet/core/blazor/security.
+
+If adding Blazor Identity to a project with existing Identity, ensure the following changes in your Program.cs file:
```

Comment:
How about the following instead for this line?

"If the project had identity support prior to scaffolding, ensure that the following changes are present in `Program.cs`"

Reply:
soudns good